File: internal/analyzer/testdata/python/bad_signatures.py
from temporalio import workflow, activity


@workflow.defn
class BadSignatureWorkflow:
    @workflow.run
    async def run(self, name: str, age: int, active: bool) -> str:
        return f"{name}-{age}"


@activity.defn
async def bad_activity(name: str, age: int, count: int) -> str:
    return f"{name}-{age}-{count}"

File: internal/analyzer/testdata/java/BadSignatures.java
package com.example.workflows;

import io.temporal.workflow.WorkflowInterface;
import io.temporal.workflow.WorkflowMethod;
import io.temporal.activity.ActivityInterface;
import io.temporal.activity.ActivityMethod;

@WorkflowInterface
public interface BadSignaturesWorkflow {
    @WorkflowMethod
    String run(String name, int age, boolean active);
}

@ActivityInterface
public interface BadSignaturesActivity {
    @ActivityMethod
    String process(String id, int count);
}
